native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -33,27 +33,42 @@ const underlayColor = Platform.select({
 // These should probably be their own tokens.
 // For now they are derived from existing tokens.
 const largeHeight = tokens.spacingSm * 12;
-const buttonBorderWidth = tokens.borderSizeLg;
+const buttonBorderWidth = Platform.OS === 'android' ? tokens.borderSizeSm : tokens.borderSizeLg;
 
 // The base styles that are initially applied to all buttons.
 const base = StyleSheet.create({
 
-  // Applied to the outer LinearGradient element.
-  container: {
-    borderRadius: tokens.borderRadiusPill,
-    height: tokens.spacingXl,
-  },
-
-  // Applied to the TouchableHighlight element.
-  button: {
-    borderRadius: tokens.borderRadiusPill,
-    height: tokens.spacingXl,
-    paddingTop: tokens.spacingMd,
-    paddingBottom: tokens.spacingMd,
-    paddingLeft: tokens.spacingSm * 3,
-    paddingRight: tokens.spacingSm * 3,
-  },
-
+  // Applied to the outer LinearGradient or View element.
+  container: Platform.select({
+    ios: () => ({
+      borderRadius: tokens.borderRadiusPill,
+      height: tokens.spacingXl,
+    }),
+    android: () => ({
+      borderRadius: tokens.borderRadiusPill,
+      height: tokens.spacingXl + tokens.spacingSm,
+    }),
+  })(),
+
+  // Applied to the TouchableHighlight/TouchableNativeFeedback element.
+  button: Platform.select({
+    ios: () => ({
+      borderRadius: tokens.borderRadiusPill,
+      height: tokens.spacingXl,
+      paddingTop: tokens.spacingMd,
+      paddingBottom: tokens.spacingMd,
+      paddingLeft: tokens.spacingSm * 3,
+      paddingRight: tokens.spacingSm * 3,
+    }),
+    android: () => ({
+      borderRadius: tokens.borderRadiusPill,
+      height: tokens.spacingXl + tokens.spacingSm,
+      paddingTop: tokens.spacingMd,
+      paddingBottom: tokens.spacingMd,
+      paddingLeft: tokens.spacingSm * 3,
+      paddingRight: tokens.spacingSm * 3,
```

Comment:
Theres a lot of room to use the button tokens as the layer of indirection here. I don't know what we've done previosuly for web with values that are multiple of tokens as we cannot express that in Theo. @jamesf3rguson would you know?

Reply:
we don't have button tokens, I can create an alias for it and move this logic away, although the iOS style is using the same logic/syntax, so I think it's best to do a refactor for both in a separate PR